library/src/scripts/headers/titleBarStyles.ts

```diff
@@ -205,6 +205,10 @@ export const titleBarVariables = useThemeCache(() => {
         tablet: {},
     });
 
+    const mobileLogo = makeThemeVars("mobileLogo", {
+        align: "center",
```

Comment:
Since you're using flexbox here and not text-align, I think it would be clearer to use: "justifyContent" as a variable name. I gave the same feedback to Chris for the nav alignment, so I think it would be better.

Reply:
@slafleche I disagree entirely here. Just because we use flex for alignment today, doesn't mean that variable only directly translates to flex property. It may be used in multiple classes, or we may switch away from using flex for the alignment in the future.

Reply:
From the perspective of someone utililizing the variables, I shouldn't actually care whether it's flexbox or not.